test/coord_sys/proj4_test.rb

```diff
@@ -31,15 +31,22 @@ def test_identity_transform
   end
 
   def test_simple_mercator_transform
+    # Reference transformation created with PostGIS
+    #
+    # WGS84 to WGS 84 / Pseudo-Mercator:
+    # SELECT ST_AsText(ST_Transform(ST_SetSRID(ST_MakePoint(degrees(0.01), degrees(0.01)), 4326), 3857));
+    #
+    # WGS 84 / Pseudo-Mercator to WGS84:
+    # SELECT radians(ST_X(point)), radians(ST_Y(point)) FROM (SELECT ST_Transform(ST_SetSRID(ST_MakePoint(-20000000, -20000000), 3857), 4326) AS point) p;
     geography = RGeo::CoordSys::Proj4.create("+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs", radians: true)
     projection = RGeo::CoordSys::Proj4.create("+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0 +k=1.0 +units=m +nadgrids=@null +wktext +no_defs")
-    assert_xy_close(project_merc(0, 0), RGeo::CoordSys::Proj4.transform_coords(geography, projection, 0, 0, nil))
-    assert_xy_close(project_merc(0.01, 0.01), RGeo::CoordSys::Proj4.transform_coords(geography, projection, 0.01, 0.01, nil))
-    assert_xy_close(project_merc(1, 1), RGeo::CoordSys::Proj4.transform_coords(geography, projection, 1, 1, nil))
-    assert_xy_close(project_merc(-1, -1), RGeo::CoordSys::Proj4.transform_coords(geography, projection, -1, -1, nil))
-    assert_xy_close(unproject_merc(0, 0), RGeo::CoordSys::Proj4.transform_coords(projection, geography, 0, 0, nil))
-    assert_xy_close(unproject_merc(10_000, 10_000), RGeo::CoordSys::Proj4.transform_coords(projection, geography, 10_000, 10_000, nil))
-    assert_xy_close(unproject_merc(-20_000_000, -20_000_000), RGeo::CoordSys::Proj4.transform_coords(projection, geography, -20_000_000, -20_000_000, nil))
+    assert_xy_close([0, 0], RGeo::CoordSys::Proj4.transform_coords(geography, projection, 0, 0, nil))
```

Comment:
Looks like this assertion is failing all of the tests on the ci runner. Any thoughts on why?